azure_li_services/units/install.py

```diff
@@ -41,41 +41,37 @@ def main():
     if packages_config:
         install_source = Defaults.mount_config_source()
 
-        try:
-            local_repos = {}
-            local_repos.update(
-                import_raw_sources(packages_config, install_source)
+        local_repos = {}
+        local_repos.update(
+            import_raw_sources(packages_config, install_source)
+        )
+        local_repos.update(
+            import_repository_sources(packages_config, install_source)
+        )
+        for repository_name, repository_metadata in local_repos.items():
+            repository_location = repository_metadata[0]
+            Command.run(
+                [
+                    'zypper', 'removerepo', repository_name
+                ], raise_on_error=False
```

Comment:
rasie_on_error=False is the default which we take advantage of in other places, why set it explicitly here?

Reply:
nope raise_on_error=True is the default, see the Command class (command.py)

```
@classmethod
    def run(self, command, custom_env=None, raise_on_error=True):
```

The reason why we don't fail on the removerepo is to keep the process of setting up the repo running. Before we add a repo we try to remove it to get a clean state. However that removal is allowed to fail. The subsequent addrepo is not allowed to fail though. See units/install.py and the way removerepo/addrepo is called

Reply:
Thanks, I looked at command.py before writing the comment, obviously my eyes and my brain were in disagreement.